ui/src/app/pages/analysis/notebook-list.tsx

```diff
@@ -36,23 +50,68 @@ export const NotebookList = withCurrentWorkspace()(class extends React.Component
     notebookList: FileDetail[],
     notebookNameList: string[],
     creating: boolean,
-    loading: boolean
+    loading: boolean,
+    delay: number,
```

Comment:
is this intended to support other kinds of delay, or is this strictly for notebook transfer delay?

Reply:
Call to the api delay, i will change the name